lib/rubocop/cop/style/unless_minimize_negations.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Minimizes the number of negations in an `unless` using De Morgan’s laws.
```

Comment:
Why in `unless` only? Seems to me this should apply to all conditionals.

Reply:
More often people avoid `unless` than use it in difficult conditions. 
```
# I don't think the corrected version is much better. Can be seen as a double negation.
do_something if x && !y && !z
do_something unless !x || y || z

# But such a case is not bad.
do_something if !x || !y || !z
do_something unless x && y && z
```

For `if` I would make a parameter (disabled by default). But, this case is related to other cops. `Style/UnlessElse` and `Style/UnlessLogicalOperators` with `forbid_logical_operators`.

Reply:
I see. I'd be fine to keeping this limited to unless but the rationale needs to be explained clearly in the cop and it's relation to existing cops.

Reply:
my 2c: I like the idea of conditionally supporting both `if` and `unless`. Perhaps the cop name could be just `Style/MinimizeNegations` with an option to target if/unless/both?

Reply:
I'll finalize it as soon as I have time.